.travis.yml

```diff
@@ -23,9 +23,6 @@ jobs:
         - cargo fetch --verbose
         - cargo build
         - cd test && yarn
-        - cd src/sdk/ && yarn
```

Comment:
Could you add a brief description of why we added these lines before and removed them again?